maven/maven.bzl

```diff
@@ -396,6 +439,13 @@ def maven_repository_specification(
         poms.append(fetch_repo.pom_target(artifact))
 
         sha256 = properties.get(artifact_config_properties.SHA256)
+
+        # TODO: Make this a more generalized exclusion list (and move
```

Comment:
Seems that something cut out this comment.

Reply:
Woops. Meant to delete that.